Source/JavaScriptCore/wasm/WasmStreamingCompiler.cpp

```diff
@@ -172,7 +173,9 @@ void StreamingCompiler::didComplete()
     }
 
     case CompilerMode::FullCompile: {
-        m_vm.deferredWorkTimer->scheduleWorkSoon(ticket, [result = WTFMove(result)](DeferredWorkTimer::Ticket ticket) mutable {
+        auto taintedness = m_source.provider()->sourceTaintedOrigin();
+        URL url = URL(m_source.provider()->sourceURL());
```

Comment:
I wonder if we need to do `isolatedCopy` since URL is holding string, so we cannot move one to the other thread.